src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -118,8 +121,17 @@ protected ActionResult execute() {
         }
         
         List<InstructorAttributes> instructorList = logic.getInstructorsForCourse(courseId);
-        String joinLink = logic.sendJoinLinkToNewInstructor(instructorList.get(0), data.instructorShortName,
-                                                            data.instructorInstitution);
+        String joinLink = Config.getAppUrl(Const.ActionURIs.INSTRUCTOR_COURSE_JOIN)
+                                .withRegistrationKey(StringHelper.encrypt(instructorList.get(0).key))
+                                .withInstructorInstitution(data.instructorInstitution)
+                                .toAbsoluteString();
+        EmailWrapper email = new EmailGenerator().generateNewInstructorAccountJoinEmail(
+                instructorList.get(0).email, data.instructorShortName, joinLink);
+        try {
+            emailSender.sendEmail(email);
```

Comment:
Hey @wkurniawan07, I was just wondering if there was any guideline as to when we send the email through `sendEmail`, and when we should schedule the sending of emails. Would there be an issue if we send the email directly here even if there are many send email tasks in the queues waiting to be executed?